google_shopping.js

```diff
@@ -2,4 +2,43 @@ var data = require("./products.json")
 
 // Write your solutions below
 
-console.log(data["items"]);
\ No newline at end of file
+shoppingproductArray = [];
+backorderArray = [];
+multiimageArray = [];
+canonproductsArray = [];
+ebaycanonArray= [];
+for (var i = 0; i < data['items'].length; i++){
+	if (data['items'][i].kind === 'shopping#product'){
+		shoppingproductArray.push("item "+ [i]);
+	}
+	if (data['items'][i].product.inventories[0].availability === 'backorder'){
+		backorderArray.push("item "+ [i]);
+	}
+	if (data['items'][i].product.images.length > 1){
+		multiimageArray.push("item "+ [i]);
+	}
+	if (data['items'][i].product.brand === 'Canon'){
+		canonproductsArray.push("item "+ [i]);
+		if (data['items'][i].product.author.name.includes('eBay')){
```

Comment:
Nice job with .includes()! I'm a fan of this as well, but keep in mind it's not supported in all browsers. An alternative method is using .indexOf():

```if (data['items'][i].product.author.name.indexOf('eBay') !== -1){```